src/code42cli/cmds/users.py

```diff
@@ -251,7 +253,7 @@ def remove_alias(state, username, alias):
 @click.argument("username")
 @sdk_options()
 def list_aliases(state, username):
-    """List the cloud aliases for a given user."""
+    """List the cloud aliases for a given user.  Each user has a default cloud alias of their Code42 username with up to two additional aliases."""
```

Comment:
I'm pretty sure it's a _total_ of two aliases, including the code42 username. So unless they remove the code42 username as an alias (possible to do, but not sure of any use case for it), they can only add one additional alias.

Reply:
I was also a little confused by this, but based on my own testing of the API calls while I was adding these commands, it seems to be 2 + the username?

Reply:
`(venv) tora.kozic@C02G1662ML85 code42cli % code42 users add-alias tora.kozic+workato@code42.com mynewalias
Error: Cloud alias limit exceeded. A max of 2 cloud aliases are allowed.
(venv) tora.kozic@C02G1662ML85 code42cli % code42 users list-aliases tora.kozic+workato@code42.com
['torakozic', 'tora.kozic', 'tora.kozic@code42.com']
`

Reply:
It only lets me add one. Maybe you had removed the username from the one it let you add 2 more to?

Reply:
odd, maybe the new user risk profile service behaves differently, and now that detectionlists are just passing through to the new service it's only letting me add one now?

Reply:
Huh, weird.  I'll look into it and adjust the docs

Reply:
So it definitely is what you were saying, but I have no idea why I can add 3 to myself